docs/externalnode.md

```diff
@@ -0,0 +1,309 @@
+# ExternalNode
+
+## Table of Contents
+<!-- toc -->
+- [What is ExternalNode?](#what-is-externalnode)
+- [Prerequisites](#prerequisites)
+- [The ExternalNode resource](#the-externalnode-resource)
+  - [Interfaces](#interfaces)
+- [Apply Antrea NetworkPolicy to ExternalNode](#apply-antrea-networkpolicy-to-externalnode)
+- [Traffic model](#traffic-model)
+  - [OVS pair ports](#ovs-pair-ports)
+  - [OpenFlow pipeline](#openflow-pipeline)
+    - [Non-IP packet](#non-ip-packet)
+    - [IP packet](#ip-packet)
+- [RBAC](#rbac)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## What is ExternalNode?
+
+`ExternalNode` is a CRD API that manages the network connectivity and security
+on a virtual machine or a bare-metal server which is not a Kubernetes Node.
+It supports specifying which network interfaces on the external Node are expected
+to be guarded with Antrea NetworkPolicy rules. The virtual machine or bare-metal
+server presented by `ExternalNode` can be either Linux or Windows. "external Node"
+will be used to represent such a virtual machine or a bare-metal server in this
```

Comment:
This document will use Virtual Machine as an example for ExternalNode representation.?
Since bare-metal is not supported, lets use Virtual Machine as an example.